frontends/web/src/locales/en/app.json

```diff
@@ -28,6 +28,7 @@
       "xpubWrongNet": "Extended Public Key not valid for the selected account."
     },
     "extendedPublicKey": "Extended Public Key",
+    "ethAddress": "Ethereum Address",
```

Comment:
Maybe just `"address": "Address"`, so the text/translations can be reused for future Bitcoin/Litecoin/... addresses. The dropdown already makes it clear that it's Ethereum.